src/tests/integration/id/profile/id-list-profile-dates.test.ts

```diff
@@ -0,0 +1,65 @@
+import { describe, expect, afterAll } from 'vitest'
+import { ID } from '../../../../id'
+import { testWithIDData } from '../test-with-id-data'
+import { ProfileDatesAttribute } from '../../../../types'
+import simpleOmneoRequest from '../../../lib/simple-omneo-request'
+import randomString from '../../../lib/string/random'
+
+const CREATED_DATES_HANDLES : string[] = []
+const getHandle = () => { return `sdk_unit_test_dates_${randomString(5).toLowerCase()}_${Math.floor(Date.now() / 1000)}` }
+const testProfileID = process.env.OMNEO_TEST_PROFILE_ID as string
+
+describe('ID Dates', () => {
+  testWithIDData('ID SDK can get dates', async ({ IDData }) => {
+    const { tokenData } = IDData
+    const payload: ProfileDatesAttribute = {
+      name: 'Wedding',
+      date: '2024-12-02',
+      handle: getHandle(),
+      is_recurring: true,
+      recurring_schedule: 'Yearly',
+      note: 'test notes for id',
+      relationship: 'Me',
+      description: 'test description for id'
+    }
+    await simpleOmneoRequest('PUT', `/profiles/${testProfileID}`, {
```

Comment:
could you please use the response of the simpleOmneoRequest to populate your CREATE_DATES_HANDLES
So in the event the IDClient.profile.getDates() test fails, it can still clean itself up

Reply:
changed.